packages/create-pantheon-decoupled-kit/src/generators/next-wp-search-api-addon.generator.ts

```diff
@@ -0,0 +1,38 @@
+import chalk from 'chalk';
+import {
+	addWithDiff,
+	convertCSSModules,
+	runInstall,
+	runLint,
+} from '../actions';
+import type { BaseGeneratorData, DecoupledKitGenerator } from '../types';
+import { outDirPrompt, tailwindcssPrompt } from '../utils/sharedPrompts';
+
+interface NextWpSearchApiAddonAnswers extends BaseGeneratorData {
+	search: boolean;
+}
+
+export const nextWpSearchApiAddon: DecoupledKitGenerator<NextWpSearchApiAddonAnswers> =
```

Comment:
This should be in `types.ts` and the name should be `Data` not `Answers`. The base answers still need to be passed in to the generic type.

Also, it should be moved to `types.ts` so that it can be added to `InputIndex`. Right now `Input` aka `data` doesn't know about the `search` or `cmsType` prop.